lookout/style/format/visualizer/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3"
```

Comment:
This is a docker compose config while we need a pod definition. I suppose that this file is removed in the next PR.